Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM clojure:lein-2.9.1 AS BASE
+
+RUN apt-get update
+RUN apt-get install --no-install-recommends -yy curl unzip build-essential zlib1g-dev
+WORKDIR "/opt"
+RUN curl -sLO https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-20.3.0/graalvm-ce-java11-linux-amd64-20.3.0.tar.gz
```

Comment:
Can't we use directly https://hub.docker.com/r/oracle/graalvm-ce/tags instead of manually downloading?

Reply:
Maybe, but we would need to download and setup lein instead so the tradeoff is not really clear.